index.html

```diff
@@ -13,53 +13,162 @@
-            <li class="hour"><img src="images/↳ Time.jpg" alt=""></li>
-            <li class="wifi"><img src="images/Wifi.jpg" alt=""></li>
-            <li class="menuu"><img src="images/Union.jpg" alt=""></li>
-        </ul>
-
-    </div>
-        
-    <header class="logo">My Logo
-    </header>
-
-    <br>
-    <br>
-    <div class="contenedor">
         <header>
-            <h1>Im Lucca Glad to see you</h1>
+            <nav class="content">
+                <ul>
+                    <li class="menu"><img src="images/mennu.PNG" alt=""></li>
+                </ul>
+            <a class="logo" href="index.html"> <img src="images/MyLogo.jpg" alt=""></a>
+            </nav>   
         </header>
-    </div>
-    <br>
-
-    <section class="main">
-        <p>
-            Im a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        <div class="contenedor">
+        <header>
+            <h1>Im Lucca
+                 Glad to see
+                  you!</h1>
+        </header>
+        </div>
+        <div class="main"><p>
+        Im a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
         </p>
+        </div>
+        <p class="let"><img src="images/LETSCONNECT.jpg" alt=""></p>
+        <div>
+            <ul>
+                <li> <a href="https://twitter.com/LuccaMicale_"><img class="twitter" src="images/Vectortw.jpg" alt=""></a></li>
+                <li> <a href="#"><img class="letteri" src="images/VectorI.png" alt=""></a></li>
+                <li> <a href="#"><img class="lettern" src="images/Vectorn.jpg" alt=""></a></li>
+                <li> <a href="#"><img class="letterm" src="images/Vectorm.jpg" alt=""></a></li>
+                <li> <a href="https://github.com/luccamicale"><img class="git" src="images/Vectorgit.jpg" alt=""></a></li>
+                <li> <a href=""><img class="hand" src="images/Vectorhnad.jpg" alt=""></a></li>
+            </ul>
+        </div>
+    </section>
+
+    <section class="all1">
+        <img class="one" src="images/SnapshootPortfolio(2).jpg" alt="">
+        <h2 class="tonic">Tonic</h2>
+        <div class="list">
+            <ul class="inline">
+                <li><img class="words" src="images/Frame2.jpg" alt=""></li>
+            </ul>
+        </div>
+        <h3 class="subtitle">A daily selection is privaly 
+            personalized reads; 
+            no accounts or sign-ups required
+        </h3>
+        <div class="leng">
+            <ul>
+                <li><img class="lenguage" src="images/Tags.jpg" alt=""></li>
+                <li><a href="https://github.com/luccamicale"><img class="see" src="images/Enabled.jpg" alt=""></a></li>
```

Comment:
Great job @luccamicale 🤝

- [ ] I would suggest you create a parent section where you may put all individual work sections.
- [ ] As specified in the project requirements, the contents of the works section should be aligned with `display: grid`.

Reply:
Kindly, I suggest you follow the advice of the previous reviewer, it will make your website look and work more PRO 😉